### PR TITLE
feat: lint privacy policy

### PR DESCRIPTION
**Pull request by wldeh**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have updated the [CHANGELOG](https://github.com/scribe-org/Scribe-iOS/blob/main/CHANGELOG.md) with a description of my changes for the upcoming release (if necessary) 

---

### Description

This PR updates the privacy policy by improving grammar and clarity.

### Related issue

- #50

### CODE REVIEW

File: PRIVACY.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@ This SERVICE makes use of third-party data. All data used in the
 creation of this SERVICE comes from sources that allow its full use
 in the manner done so by the SERVICE. Specifically, the data for this
 SERVICE comes from Wikidata and Wikipedia. Wikidata states that, "All
-structured data in the main, property and lexeme namespaces is made
+structured data in the main, property and lexeme namespaces are made
```
Comment (andrewtavis) [inline PRIVACY.txt:40]: Checking this now :) This is a direct quote, so it should remain as is. As data is "uncountable" even though obviously we should count it, I think that "is" is also correct here :)

**Comment by andrewtavis** [comment]
Thanks for the efforts so far, @wldeh! 🚀

File: PRIVACY.txt (hunk 1 above)
Comment (wldeh) [reply PRIVACY.txt:40]: Got it! I'll keep the quote as is. Thanks for the feedback.

**Comment by andrewtavis** [comment]
Thanks, @wldeh! Will check this later in the day after work 😊

**Review by andrewtavis — APPROVED** [review]
Thank you, @wldeh! I'll extend these changes to the `.swift` file for the app as well as the German version 😊

**Pull request merged**